SampleUsages/TestScenario.cs

```diff
@@ -34,56 +37,51 @@ internal void RunScenario(ILogger logger, ServerlessResultManager.TestScenario d
             var scenarioType = Tuple.Create(this.Platform, this.TriggerType);
             FunctionTest test = null;
             TriggerTestLoadProfile profile;
-            var inputCount = 0;
+
+            string[] inputItems;
+            try
+            {
+                inputItems = this.Input ?? Directory.GetFiles(this.InputPath);
+            }
+            catch
```

Comment:
Actually I think I'm just confused about this code. Why the call to Directory.GetFiles(), what scenario is that for?